From 2b16cb8d7968988d6e5bffa5adc8ffbe636e1d65 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Tue, 14 Apr 2020 20:49:46 +0200
Subject: [PATCH] scripts/opkg-make-index: add SHA256Sum

This adds SHA256Sum as Turris updater in some version was expecting this
string instead of SHA256sum.
---
 scripts/ipkg-make-index.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/ipkg-make-index.sh b/scripts/ipkg-make-index.sh
index 910e987..4aeba15 100755
--- a/scripts/ipkg-make-index.sh
+++ b/scripts/ipkg-make-index.sh
@@ -22,6 +22,7 @@ for pkg in `find $pkg_dir -name '*.ipk' | sort`; do
 	tar -xzOf $pkg ./control.tar.gz | tar xzOf - ./control | sed -e "s/^Description:/Filename: $sed_safe_pkg\\
 Size: $file_size\\
 SHA256sum: $sha256sum\\
+SHA256Sum: $sha256sum\\
 Description:/"
 	echo ""
 done
-- 
2.26.0

